.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Describe your changes
+Include photos if necessary.
+
+## Checklist
+- [ ] Run `yarn format` over your changes
+- [ ] Ensure `yarn lint` and `yarn tsc` checks are passing
+- [ ] Add necessary tests
+- [ ] Add Changeset
+- [ ] Write documentation (README.md)
```

Comment:
Should these 2 items be moved under For new components... ?

Reply:
Changesets and documentation should also be implemented for any component updates :)